<?xml version="1.0" encoding="UTF-8"?>
<!--
     Okno pre editaciu objektov na scene.
     Autori:
       Lisik Miroslav <xlisik00@stud.fit.vutbr.cz>
       Pokorny Fridolin <fridex.devel@gmail.com>
-->
<ui version="4.0">
 <class>PNObjectWindow</class>
 <widget class="QDialog" name="PNObjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNObject Edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTokens">
         <property name="text">
          <string>Tokens:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCondition">
         <property name="text">
          <string>Condition:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMode">
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPriority">
         <property name="text">
          <string>Priority:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="tokensEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="conditionEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="modeEdit"/>
       </item>
       <item>
        <widget class="QSpinBox" name="priorityBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PNObjectWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PNObjectWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
